src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,114 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.dukeexception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.tasklist.TaskList;
+
+
+/**
+ * class that handles saving and reading tasks saved on the local hard disk
+ */
+public class Storage {
+    private File allTasks;
+    private File taskFolder;
+
+    /**
+     * Constructor
+     * @param filePath A string representing the location of the local file containing all tasks
+     * @param folderPath A string representing the location of the folder containing the above file
+     */
+    public Storage(String filePath, String folderPath) {
+        this.allTasks = new File(filePath);
+        this.taskFolder = new File(folderPath);
+    }
+
+    /**
+     * a method that loads the local tasks into the TaskList if they exist. prints customised
+     * messages if the file and/or folder does not exist, and create them accordingly
+     * @return an empty ArrayList of Tasks if no local tasks are found, or an ArrayList
+     *         containing all local tasks if they are found.
+     */
```

Comment:
You might want to add a @throws comment in this jdoc, 
```suggestion
    /**
     * a method that loads the local tasks into the TaskList if they exist. prints customised
     * messages if the file and/or folder does not exist, and create them accordingly
     * @return an empty ArrayList of Tasks if no local tasks are found, or an ArrayList
     *         containing all local tasks if they are found.
     * @throws ...
     */
```